### DISCUSSION TITLE
Latest Update v 0.13.1 - 500 Server Error

### DISCUSSION THREAD

**Discussion by Ja50n-Kru** [alexjustesen/speedtest-tracker#851] [Q&A] [answered]
@alexjustesen Hi, I just updated to v0.13.1 and immediately cannot access the dashboard. I get 500 server error. Looking at the logs, the container starts up ok, but I get this error as the first log after I try to access the dashboard:

[2023-10-04 22:52:54] production.ERROR: No application encryption key has been specified. {"exception":"[object] (Illuminate\\Encryption\\MissingAppKeyException(code: 0): No application encryption key has been specified. at /var/www/html/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:79)
22:52:54
[stacktrace]

It does seem to be running, though, as a test was pushed to my influxdb. 

Is it just me, or an issue with the latest build? 

V0.13.0 was working perfectly before. Any tips to get back up and running?

**Comment by alexjustesen** [comment, marked as answer]
Do a search in issues for app key or check out the docs. FAQ has how to resolve this issue.

**Reply by Ja50n-Kru** [reply]
Thanks. I did multiple searches on my phone and nothing came up - I must have had a filter on somehow! Setting the persistent key solved it.

**Reply by masterwishx** [reply]
.env file data is not readed correctly!
adding same APP_KEY from .env to Docker build as Var Fix the problem.
Also key of telegram is not working from .env file

**Reply by masterwishx** [reply]
@alexjustesen 
it seems like all data from .env file is not readed at all in latest version
like APP_KEY, telegram,Email .
all this data now should be added as ENV vars in docker or will be fixed?

**Comment by mircotronchin** [comment]
After this morning update I have the same problem: 500 server error.
My installation use sqlite and I would like to maintain it.
what I've to do to resolve this problem?
thank you.

**Reply by mircotronchin** [reply]
resolved now with this procedure:
docker exec -ti speedtest-tracker /bin/bash
php artisan key:generate --show
copy the result and redeploy the container vith this env variable: -e APP_KEY=base64:24iYXcWXDoC......... note: base64 must be included. 

tks.

**Reply by Shotblocker** [reply]
After these steps I receive the error: Unsupported cipher or incorrect key length when looking in debug mode.

**Reply by Ja50n-Kru** [reply]
You need to do as the FAQ says. Paste the response to the command in the APP Key variable.

**Reply by Shotblocker** [reply]
For those who faced the same issue as me the "base64:" needs to be included not just the hash.

**Reply by benr916** [reply]
APP_KEY environment variable added to the Docker Compose file using the value read from the .env file fixed the 500 server error for me. Thank you!
